### PR TITLE
virtualbox 4.3.28 cask

### PR DESCRIPTION
**Pull request by mikeolivieri**
cask for latest version of virtual box

### CODE REVIEW

**Comment by fromonesrc** [comment]
Looks :thumbsup: to me, @articulate/web-ops can I get a second opinion?

File: Casks/virtualbox4328100309.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'virtualbox4328100309' do
+  version '4.3.28-100309'
+  sha256 '16ef55298abe5de8cd0e9ac88895d6404cde5a17d9aa126f8860c411c4eeeac2'
+
+  url "http://download.virtualbox.org/virtualbox/#{version.sub(/-.*$/, '')}/VirtualBox-#{version}-OSX.dmg"
+  homepage 'http://www.virtualbox.org'
+  license :gpl
+
+  pkg 'VirtualBox.pkg'
+  binary '/Applications/VirtualBox.app/Contents/MacOS/VBoxManage'
```
Comment (Shadowssong) [inline Casks/virtualbox4328100309.rb]: I don't see the `binary` setting in the other virtualbox cask, does this let cask know to check for any running processes when doing uninstall?

Comment (mikeolivieri) [inline Casks/virtualbox4328100309.rb]: No, these are some command line management tools that will get symlinked. If you run a regular `brew cask install virtualbox` you can see what actually happens. 

Taken from here: https://github.com/caskroom/homebrew-cask/blob/9b2eaab126d221e02b82ecdf9ae1002e2870770b/Casks/virtualbox.rb

Comment (fromonesrc) [inline Casks/virtualbox4328100309.rb]: Any reason not to just use that upstream version instead of maintaining our own cask for that?

Comment (mikeolivieri) [inline Casks/virtualbox4328100309.rb]: No idea. I was just following the pattern :v:

Comment (fromonesrc) [inline Casks/virtualbox4328100309.rb]: @Shadowssong what do you think about that?

Comment (mikeolivieri) [inline Casks/virtualbox4328100309.rb]: Well one issue would be that whenever the upstream cask was updated, people would have to stop their vms before updating boot. not sure if that is a real concern though. thoughts?

Comment (fromonesrc) [inline Casks/virtualbox4328100309.rb]: Yeah that would not be so great. I'm :thumbsup: on leaving this here and :ship:

Comment (Shadowssong) [inline Casks/virtualbox4328100309.rb]: Yep it's not versioned on the main cask repo.  Sounds good :ship:

**Pull request merged**